Lab4/WebContent/admin/create-food.jsp

```diff
@@ -0,0 +1,107 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8"%>
+      <%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+      
+<%--           <%@ taglib uri="adminHeader.tld" prefix= "cs3220"%>
+ --%>      
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<title>Insert title here</title>
+  <link rel="stylesheet" type="text/css" href="../app.css">
+    <link href='http://fonts.googleapis.com/css?family=Lobster' rel='stylesheet' type='text/css'>
+    <link href='http://fonts.googleapis.com/css?family=Cabin' rel='stylesheet' type='text/css'>
+    <meta name="viewport" content="width=device-width, initial-scale=1" shrink-to-fit="no ">
+
+</head>
+<body>
+	    <header class="clearfix ">
+        <nav>
+          
+
+            <a href="create-food.jsp ">Create Food Item</a>
+            
+
+            <a href="inventory.jsp ">Inventory</a>
+            
+             <a href="order-statuses.jsp">Order Status</a>
```

Comment:
all these should be done in Servlet mapping and not going to jsp directly. See technical requirement.